staff/joseluis-juste/cv/Index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículum Vitae</title>
+</head>
+
+<body>
+        <fieldset>
```

Comment:
Maybe `<fieldset>` is not the most accurated tag in this case, because dispite you are using `<legend>` and `<label>` the prupouse is not the creation of a form. As far as I know`<fieldset>` was desingned to handle forms.

Firther information:

- [MDN Spanish Recource](https://developer.mozilla.org/es/docs/Web/HTML/Elemento/fieldset)

- [MDN HTML Tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)